subprojects/core/src/main/java/org/gradle/api/internal/StartParameterInternal.java

```diff
@@ -44,6 +45,13 @@
     private boolean searchUpwards = true;
     private boolean useEmptySettings = false;
 
+    public StartParameterInternal() {
+    }
```

Comment:
:question: Should this call `super()`?

Reply:
The `super()` call is implicit in this case, it can be made explicit but I didn't think it would improve readability.

Reply:
Right. No strong opinion